src/AerosolActivation.jl

```diff
@@ -0,0 +1,290 @@
+"""
+    Aerosol activation scheme, which includes:
+
+  - mean hygroscopicity for each mode of the aerosol size distribution
+  - critical supersaturation for each mode of the aerosol size distribution
+  - maximum supersaturation
+  - total number of particles actived
+  - total mass of particles actived
+"""
+module AerosolActivation
+
+using SpecialFunctions
+
+using Thermodynamics
+
+using CloudMicrophysics.AerosolModel
+using CloudMicrophysics.Microphysics_1M: G_func
+
+using CLIMAParameters
+using CLIMAParameters: gas_constant
+using CLIMAParameters.Planet:
+    ρ_cloud_liq,
+    R_v,
+    grav,
+    molmass_water,
+    molmass_dryair,
+    cp_d,
+    surface_tension_coeff
+using CLIMAParameters.Atmos.Microphysics: K_therm, D_vapor
+
+const APS = AbstractParameterSet
```

Comment:
Maybe not a blocker, but one pattern I'm starting to try to follow more is qualifying inputs by doing, for example,
```julia
import Thermodynamics
TD = Thermodynamics
```
And later calling methods with `TD.air_temperature(ts)`. I know that this introduces a bunch of aliases that need to be tracked, but it helps us to easily find where methods are defined. I've started doing the same with CLIMAParameters in TC.jl.

Reply:
Sounds reasonable. I'll fix it in the next PR for the whole package. Since we would have to do it in the microphysics modules as well. And this PR is already big